website/src/documentation/dsls/sql/create-external-table.md

```diff
@@ -19,20 +21,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# CREATE TABLE
+# CREATE EXTERNAL TABLE
 
-Beam SQL's `CREATE TABLE` statement registers a virtual table that maps to an
+Beam SQL's `CREATE EXTERNAL TABLE` statement registers a virtual table that maps to an
 [external storage system](https://beam.apache.org/documentation/io/built-in/).
```

Comment:
Should be {{ site.baseurl }}

Reply:
Thanks; I will fix these upstream in `beam-site` and then re-import